spec/unit/filter_option_parser_spec.rb

```diff
@@ -100,8 +100,8 @@
         filter = subject.parse("inspect", "skip-files" => "\\@file_with_at,/foo")
 
         expect(filter.to_array).to eq([
-          "/unmanaged_files/files/name=@file_with_at",
-          "/unmanaged_files/files/name=/foo"
+          "/unmanaged_files/name=@file_with_at",
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.